content/configuration/dev-server.md

```diff
@@ -129,6 +165,25 @@ To disable `contentBase`:
 contentBase: false
 ```
 
+Usage via the CLI
+```
+webpack-dev-server --content-base /path/to/content/dir
+```
+
+## `devServer.disableHostCheck`
```

Comment:
was this option intentionally left out of the docs to prevent people from using it? If so, I can remove this.